helm/bold-common/README.md

```diff
@@ -163,6 +178,35 @@ _See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command doc
 
 ## Application Startup
 
-Configure the Bold BI application startup to use the application. Please refer the following link for more details on configuring the application startup.
+1. Configure the Bold BI application startup to use the application. Please refer the following link for more details on configuring the application startup.
     
-https://help.boldbi.com/embedded-bi/application-startup
+   https://help.boldbi.com/embedded-bi/application-startup
+
+2. Navigate to administration configuration file.
+   ```sh
+   <DNS>/ums/administration/config-editor
+   ```
+   ![Configuration-editor](/helm/docs/images/config-edit.png)
+
+3. In product.json add below lines for Bold Reports.
+    ```console
+    {
+    "Name": "BoldReports",
+    "SetupName": "BoldReports_EnterpriseReporting",
+    "Version": "4.2.52",
+    "IDPVersion": "3.1.21",
+    "IsCommonLogin": true
+    }
+    ```
+   >**Note:** The IDP version of Bold BI or Bold Reports should be same.
+
+4. Set `IsCommonLogin` property to be `true` for Bold BI and Bold Reports.
+     ![Product Json](/helm/docs/images/reports.png)
+
+5. Refer the below document to activate License either by using your login credientials or by offline unlock key for Bold Reports.
```

Comment:
5. Refer to the document below to activate the License either by using your login credentials or by an offline unlock key for Bold Reports.

Reply:
concern addressed